services/web/app.py

```diff
@@ -178,7 +178,7 @@ def auth_saml(self, auth):
             if authorized:
                 resp = make_response("Authorized", 200)
                 resp.headers["X-RH-Identity"] = self.make_identity_header(
-                    "Associate", "saml-auth", {k: v if len(v) > 1 else v[0] for k, v in auth_data}
+                    "Associate", "saml-auth", {k: v if (len(v) > 1 or k == "Role") else v[0] for k, v in auth_data}
```

Comment:
I feel like we should have an enumeration of multi-value attributes, no?

Reply:
Happy to rebase on #12 when it goes in, but I updated this PR.